Lucerne/MsEth1.xml

```diff
@@ -48,33 +49,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         </support>
                         <extent>
-                           <measure unit="leaf">12</measure>
-                           <note>dimensions for the folded book</note>
+                           <measure unit="leaf">3</measure>
+                           <measure unit="page">27</measure>
+                           <note>3 leaves, tied together and folded to 27 pages; dimensions for the folded book</note>
                            <dimensions type="outer" unit="mm">
                               <height>52</height>
                               <width>42</width>
-                              
                            </dimensions>
                         </extent>
-                        <foliation></foliation>
-                        <condition key="good"></condition>                        
+                        <foliation>No foliation</foliation>
+                        <collation>Three slender pieces of parchment have been tied together and folded into 27 pages, the first and the last  of which
+                           are attached to the cover and left blank.</collation>
+                        <condition key="good"/>                        
                      </supportDesc>
-                     
-                  </objectDesc>                  
-                  <handDesc><handNote xml:id="h1" script="Ethiopic"></handNote></handDesc>
-                  
-                  
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="10"/>                            
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>                  
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">Second half of 17th to first half of the 20th century</seg>
+                        <seg type="ink">Black, red</seg>
+                        <seg type="rubrication">Incipits, name of the owner</seg>
+                        <note>Long and slender script.</note>
+                        <note>Ligature of <foreign xml:lang="gez">ግዚ</foreign>.</note>
+                     </handNote>
+                  </handDesc>
+                  <bindingDesc>
+                     <binding contemporary="Ethiopian">
+                        <decoNote xml:id="b1" type="Boards">Two wooden boards, tied to the parchment with a leather cord.</decoNote>    
+                        <decoNote xml:id="b2" type="bindingMaterial"><material key="wood"></material></decoNote>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
-                     
-                     <origDate notBefore="1800" notAfter="1900">nineteenth  century (?) </origDate>
+                     <origDate notBefore="1650" notAfter="1950">Second half of 17th to first half of the 20th century</origDate>
```

Comment:
The manuscript is unlikely older than the mid 19th, the dating thus is mid-19th to mid-20th century, in my opinion (preference to the more recent half), unless there is any other external evidence (like known acquisition time). the shapes of ደ and ይ show this, also in part  ግ. The scribe may try to imitate, a bit crudely, another "older" hand. The binding, on two boards, is interesting. We should try to avoid datings like 1650-1950 as they are too wide and not really helpful.

_Originally posted by @DenisNosnitsin1970 in https://github.com/BetaMasaheft/Persons/issues/769#issuecomment-1831819144_

Reply:
ok.